gui/wxpython/gui_core/forms.py

```diff
@@ -1053,11 +1053,10 @@ def __init__(self, parent, giface, task, id=wx.ID_ANY, frame=None, *args, **kwar
         elif style == 1:  # basic left
             self.notebook = FormNotebook(self, style=wx.BK_LEFT)
             self.notebook.Bind(wx.EVT_NOTEBOOK_PAGE_CHANGED, self.OnPageChange)
-        elif style == 2:  # fancy green
+        elif style == 2:  # normal style
```

Comment:
Could you completely remove this style? It doesn't make sense to leave it there, it's very similar to the simple top style.